### ISSUE TITLE
How to support Bootstrap 5 form-floating class and floatingInput label?

### ISSUE THREAD

**Issue by christobh** [heartcombo/simple_form#1768]
Good day,

Not an issue, so let me know if I should move this somewhere else?

Was just hoping someone could point out how / if it is possible to handle the following with simple form?

This is a default example of what a Devise template would look like using simple_form:

```
<%= simple_form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
  <div class="form-inputs">
    <%= f.input :email,
                required: false,
                autofocus: true,
                input_html: { autocomplete: "email" } %>
  </div>

  <div class="form-actions">
    <%= f.button :submit, "Log in" , class: 'btn btn-primary mt-2' %>
  </div>
<% end %>
```

**My question is how to make use of the form-floating class and floatingInput label sinilar to the standard from_for example below?:**

This is a default example of what a Devise template would look like using with form_for:

   <%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>

    <div class="form-floating">
      <%= f.email_field :email, autocomplete: "email", type:"email", class: "form-control", id:"floatingInput", placeholder: "name@example.com", required: false, autofocus: false %>

      <label for="floatingInput" style="color: black">Email address</label>
    </div>

    <%= f.submit "Log in" , class: "w-100 btn btn-lg btn-primary" %>

<% end %>

Any guidance / assistance would be appreciated!

**Comment by nashby** [comment]
Hey! Please use StackOverflow for asking question, we don't use issues for questions.

TL;DR You can check how it's done in https://github.com/heartcombo/simple_form-bootstrap. It's a repo with example usages of Bootstrap with SimpleForm. Here's floating label example here https://simple-form-bootstrap.herokuapp.com/examples/floating_label
